examples/src/stylefy/examples/grid.cljs

```diff
@@ -13,28 +13,28 @@
 
 (defn grid [options schema data]
   [:div (use-style grid-style)
-   [:table (use-sub-style grid-style :table)
-    [:thead (use-sub-style grid-style :thead)
+   [:table (use-style (sub-style grid-style :table))
+    [:thead (use-style (sub-style grid-style :thead))
      [:tr
       (doall
-        (map-indexed
-          (fn [column-index s]
-            ^{:key column-index}
-            [:th (use-sub-style grid-style :cell)
-             (:title s)])
-          schema))]]
+       (map-indexed
+        (fn [column-index s]
```

Comment:
Just curious to now what editor you are using? I'm using IntelliJ IDEA with Cursive and two spaces is the default formatting method. :)